ideascube/mediacenter/tests/test_remove_medias.py

```diff
@@ -0,0 +1,100 @@
+    metadata = ('title,summary,credits,path\n'
+                'my video,my video summary,BSF,a-video.mp4\n')
+    write_metadata(metadata)
+    call_command('import_medias', CSV_PATH)
+    original_path = os.path.join(settings.MEDIA_ROOT, 'mediacenter/document', 'a-video.mp4')
+    assert os.path.exists(original_path)
+    call_command('remove_medias', 'clean_orphans')
+    assert os.path.exists(original_path)
+
+
+def test_clean_orphans_should_correctly_remove_ophan_files(settings):
+    metadata = ('title,summary,credits,path,preview\n'
+                'my video,my video summary,BSF,a-video.mp4,an-image.jpg\n')
+    write_metadata(metadata)
+    call_command('import_medias', CSV_PATH)
+    original_path = os.path.join(settings.MEDIA_ROOT, 'mediacenter/document', 'a-video.mp4')
+    preview_path = os.path.join(settings.MEDIA_ROOT, 'mediacenter/preview', 'an-image.jpg')
+    Document.objects.all().delete()
+    assert os.path.exists(original_path)
+    assert os.path.exists(preview_path)
+    call_command('remove_medias', 'clean_orphans')
+    assert not os.path.exists(original_path)
+    assert not os.path.exists(preview_path)
+
+
+def test_clean_orphans_should_fail_nicely_if_cannot_remove(settings, capsys):
+    metadata = ('title,summary,credits,path,preview\n'
+                'my video,my video summary,BSF,a-video.mp4,an-image.jpg\n')
+    write_metadata(metadata)
+    call_command('import_medias', CSV_PATH)
+    original_path = os.path.join(settings.MEDIA_ROOT, 'mediacenter/document', 'a-video.mp4')
+    preview_path = os.path.join(settings.MEDIA_ROOT, 'mediacenter/preview', 'an-image.jpg')
+    Document.objects.all().delete()
+    assert os.path.exists(original_path)
+    assert os.path.exists(preview_path)
+    os.chmod(os.path.join(settings.MEDIA_ROOT, 'mediacenter/document'), stat.S_IRUSR | stat.S_IXUSR)
+    call_command('remove_medias', 'clean_orphans')
+    assert os.path.exists(original_path)
+    assert not os.path.exists(preview_path)
+    _, err = capsys.readouterr()
+    assert "ERROR while deleting {}".format(original_path) in err
+
+
+def test_clean_orphans_should_not_remove_ophan_files_if_dry_run(settings, capsys):
+    metadata = ('title,summary,credits,path,preview\n'
+                'my video,my video summary,BSF,a-video.mp4,an-image.jpg\n')
+    write_metadata(metadata)
+    call_command('import_medias', CSV_PATH)
+    original_path = os.path.join(settings.MEDIA_ROOT, 'mediacenter/document', 'a-video.mp4')
+    preview_path = os.path.join(settings.MEDIA_ROOT, 'mediacenter/preview', 'an-image.jpg')
+    Document.objects.all().delete()
+    assert os.path.exists(original_path)
+    assert os.path.exists(preview_path)
+    call_command('remove_medias', 'clean_orphans', dry_run=True)
+    assert os.path.exists(original_path)
+    assert os.path.exists(preview_path)
+    out, _ = capsys.readouterr()
+    assert original_path in out
+    assert preview_path in out
```

Comment:
One test that you're missing is to ensure that leftover files are removed after updating a `Document`.

In addition, it would be nice to test the whole thing on a slightly more complex case, mixing files to remove and files to keep, both in originals and previews.